Update Readme to provide better navigation throughout all community platform for discussion and find resources (#149)
filecoin-project/community: README.md

## About

Welcome to the Filecoin community repository! You can treat this repo as your go-to for all meta, non-code discussions in the [*Discussions Forum*](https://github.com/filecoin-project/community/discussions), documents used by multiple repositories (such as the Code of Conduct), and how to interact with the Filecoin project and other community members.

To get the latest network notifications, subscribe to [status.filecoin.io](https://status.filecoin.io).

If you are interested in discussing code or protocol design, feel free to come talk to us on our [forums](#forums), [chat channels](#chat), or in issues on our other [Filecoin project repos](https://github.com/filecoin-project).

## Join the Community

**> Note: Before posting to different communications channels, make sure to read the [Code of Conduct](https://github.com/filecoin-project/community/blob/master/CODE_OF_CONDUCT.md).**

> The vast majority of the Filecoin Project conversations, including implementation, community support, ecosystem new and etc, are held on:

Reviewer: ```suggestion
The vast majority of the Filecoin Project conversations, including implementation, community support, ecosystem news, etc. take place on:
```
Minor grammar tweaks.

Implementation dev teams tend to check their corresponding GitHub repo issues and discussions regularly and try to respond within 5 business days.

We also push updates to the community via:

- Forum - [`Network and implementations updates`](https://github.com/filecoin-project/community/discussions/categories/network-and-implementations-updates) updates and [`news and events`](https://github.com/filecoin-project/community/discussions/categories/news-and-events)
- Filecoin blog: [https://blog.filecoin.io](https://blog.filecoin.io)
- CryptoComputeLab blog(proof updates): [https://research.protocol.ai/groups/cryptocomputelab/](https://research.protocol.ai/groups/cryptocomputelab/)
- Youtube: [Filecoin](https://www.youtube.com/channel/UCPyYmtJYQwxM-EUyRUTp5DA)
- Twitter: [@Filecoin](https://twitter.com/Filecoin)
- WeChat ID: Filecoin-Official

![filecoin qr code](/images/qrcode_for_gh_da36751a6108_1280.jpg)

### Forums

#### Filecoin Community Forum

When in doubt or curiosity, please post in Filecoin community forum](https://github.com/filecoin-project/community/discussions) !

We love to hear what the community has to say, whether it is to:

- Ask a question

> - Gathering community feedbacks on a new feature proposal before open an FIP 

Reviewer: ```suggestion
- Gather community feedback on a new feature proposal before opening an FIP 
```
1. Gathering -> Gather to use the same tense as the rest of the list 
2. Feedback is considered an uncountable noun, so it's almost always used in the singular. (very weird quirk)
3. Open -> "opening" or "you open" (with subject), per https://www.gymglish.com/en/gymglish/english-grammar/after-and-before

- Share a new project you're working on
- Find collaborators for your own community project
- And whatever else! Honestly!

The discussion forum uses the same Code of Conduct as our other community channels. Please make sure to read this before posting.

> > Note: Chinese community forum is available [here](https://github.com/filecoin-project/community-china/discussions), which is managed by [CoinSummer](https://github.com/CoinSummer).

Reviewer: ```suggestion
> Note: A Chinese-language community forum is also available at https://github.com/filecoin-project/community-china/discussions. It is managed by [CoinSummer](https://github.com/CoinSummer).
```

Use more descriptive link text instead of "here" whenever possible, to make it easier to scan.
(Explanation: https://uxmovement.com/content/why-your-links-should-never-say-click-here/)

> #### lotus Discussion

Reviewer: ```suggestion
#### lotus Discussion Forum
```

#### Filecoin Proving Subsystem Discussion

The Filecoin Proving Subsystem (or FPS) provides the storage proofs required by the Filecoin protocol. If you have any question regard to the current proof, feedbacks for proof performance on different machines, ideas to improve proof or implementing new proof, join the [forum here](https://github.com/filecoin-project/rust-fil-proofs/discussions)!

### Implementation Related Repo

### Chat

Vast majority of community live chat is happening in [Filecoin Project Slack](https://filecoin.io/slack). Tag @fil-community-ambassadors if you have any questions!

Fun channels to join once you are in the workspace:

> - `#fil-announcement`: This channel is for official Filecoin announcements only(including network, implementations and ecosystem announcements). Subscribe to get most up-to-date news, however, please do not post questions or other messages here! 

Reviewer: ```suggestion
- `#fil-announcements`: This channel is for official Filecoin announcements only (including network, implementations, and ecosystem announcements). Join to get most up-to-date news. Please do not post questions or other messages here; they will be deleted! 
```

Channel name has an `s`

> - `_fil-lobby`: for general Filecoin related sharing mostly

Reviewer: ```suggestion
- `_fil-lobby`: for general Filecoin-related sharing
```

- `fil-help`: ask questions here if you can't find another specific channel for your question
- `fil-lotus`: for lotus related discussion
- `fil-fips`: for [Filecoin Improvement Proposals](https://github.com/filecoin-project/FIPs) related discussion 
- `fil-plus`: for Filecoin Plus([notary-governance](https://github.com/filecoin-project/notary-governance)) related discussion
- `fil-net-calibration-discuss`/`fil-net-nerpa-discuss`: for testnet disucssions

> - `fil-ecosystem-dev`: for Filecoin ecosystem projects updates and development chats

Reviewer: ```suggestion
- `fil-ecosystem-dev`: for updates and discussion about building in the Filecoin ecosystem
```

- `fil-deal-market`: promote your miner as a storage provider or find your provider as a client here

> - `hackathon-help`: join this channel if you are participating any Filecoin hackathon and have questions

Reviewer: ```suggestion
- `hackathons-help`: join this channel if you are participating any Filecoin hackathons and have questions
```

Reviewer: Channel name has an s

Please refer to [SECURITY.md](./SECURITY.md) document found in this repo on how to best report findings and participate on the bug bounty program.

## Useful links

If you are new to the Filecoin Project, below are some helpful links for you to learn more about it:
- Official website: [https://filecoin.io](https://filecoin.io)  
- [Filecoin Specification](https://spec.filecoin.io): contains documents, code, models, and diagrams that constitute the specification of the Filecoin Protocol.
- [Filecoin Docs](https://docs.filecoin.io): offers all the necessary resources to learn about Filecoin, the software and the tools to contribute to the network, either as a user looking for storage, or as a miner providing it  

## Ecosystem Projects

> If your product uses Filecoin, you're in the Filecoin ecosystem! We'd love to see all awesome projects that are built on top of the Filecoin ecosystem to be known and get used by the Filecoin community! 

Reviewer: ```suggestion
If your project uses Filecoin, you're in the Filecoin ecosystem! We'd love to see all awesome projects that are built on top of the Filecoin ecosystem to be known and get used by the Filecoin community! 
```
